CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 ## Next release
 
+## 0.41.0 / 2020-07-29
+
+* [CHANGE] configmap-reload: Update to v0.4.0 (#3334)
+* [CHANGE] Update prometheus compatibility matrix to v2.19.2 (#3316)
+* [FEATURE] Add Synthetic Probes support (#2832)
+* [FEATURE] Support Prometheus vertical compaction (#3281)
+* [ENHANCEMENT] docs: update compatibility warning (#3369)
```

Comment:
we typically only list user facing changes affecting the release artifacts

Reply:
removed :+1: